src/main/java/org/openstreetmap/atlas/checks/validation/tag/SimilarTagValueCheck.java

```diff
@@ -0,0 +1,346 @@
+            "alt_name", "destination", "name", "seamark", "turn");
+    // Common value sets that are similar but can be on the same tag
+    private static final List<List<String>> COMMON_SIMILARS_DEFAULT = List.of(
+            // Ethnonyms
+            List.of("american", "mexican"),
+            // Food
+            List.of("cafe", "cake"),
+            // Gender
+            List.of("male", "female"), List.of("woman", "man"), List.of("women", "men"),
+            List.of("male_toilet", "female_toilet"),
+            // Medical
+            List.of("radiology", "cardiology"),
+            // Sports
+            List.of("baseball", "basketball"), List.of("bowls", "boules"),
+            List.of("padel", "paddel"),
+            // Other
+            List.of("formal", "informal"), List.of("hotel", "hostel"), List.of("hump", "bump"),
+            List.of("seed", "feed"));
+
+    private final List<String> tagsToIgnore;
+    private final List<String> tagsWithSubCategoriesToIgnore;
+    private final List<List<String>> commonSimilars;
+    private final Double minValueLength;
+    private final Double minSimilarityThreshold;
+    private final Double maxSimilarityThreshold;
+
+    /**
+     * @param configuration
+     *            the JSON configuration for this check
+     */
+    public SimilarTagValueCheck(final Configuration configuration)
+    {
+        super(configuration);
+        this.commonSimilars = this.configurationValue(configuration, "filter.commonSimilars",
+                COMMON_SIMILARS_DEFAULT);
+        this.tagsToIgnore = this.configurationValue(configuration, "filter.tags",
+                TAGS_TO_IGNORE_DEFAULT);
+        this.tagsWithSubCategoriesToIgnore = this.configurationValue(configuration,
+                "filter.tagsWithSubCategories", TAGS_WITH_SUB_CATEGORIES_TO_IGNORE_DEFAULT);
+        this.minValueLength = this.configurationValue(configuration, "value.length.min",
+                MIN_VALUE_LENGTH, Double::doubleValue);
+        this.minSimilarityThreshold = this.configurationValue(configuration,
+                "similarity.threshold.min", MIN_SIMILARITY_THRESHOLD_DEFAULT, Double::doubleValue);
+        this.maxSimilarityThreshold = this.configurationValue(configuration,
+                "similarity.threshold.max", MAX_SIMILARITY_THRESHOLD_DEFAULT, Double::doubleValue);
+    }
+
+    /**
+     * Valid objects for this check objects with tags that contain multiple values.
+     *
+     * @param object
+     *            the atlas object supplied by the Atlas-Checks framework for evaluation
+     * @return {@code true} if this object should be checked
+     */
+    @Override
+    public boolean validCheckForObject(final AtlasObject object)
+    {
+        // Only want objects with tags that contain multiple values.
+        return object.getOsmTags().values().stream().anyMatch(value -> value.contains(SEMICOLON));
```

Comment:
Noting that it might be worth ensuring the semicolon is not at the end of a string, in which case we could use regex here

Reply:
Just did a comparison run against AUS data with this change
`return object.getOsmTags().values().stream().anyMatch(value -> value.contains(SEMICOLON) && (value.indexOf(SEMICOLON) != value.length() - 1));`
and the times are ~equal (59s vs 60s, both producing 1030 flags)